2015/10/25 20:26:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/25 20:27:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/25 20:27:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/25 20:27:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/25 20:27:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/25 20:27:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/25 20:28:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/25 20:28:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/25 20:28:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/25 20:28:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/25 20:28:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/25 20:28:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/25 20:29:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/project
---
2015/10/25 20:29:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/project
---
2015/10/25 20:29:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/project
---
2015/10/25 20:29:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/project
---
2015/10/25 20:31:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/25 20:31:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/25 20:31:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/25 20:31:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/25 20:32:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/25 20:32:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/25 20:32:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/project
---
2015/10/25 20:32:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/project
---
2015/10/25 20:32:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/project
---
2015/10/25 20:32:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/project
---
2015/10/25 20:56:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/25 20:56:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/25 20:56:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/25 20:56:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/25 20:56:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/25 20:56:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/25 20:58:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "causes".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('causes')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/causes
HTTP_REFERER=http://ikt-straubing.dev/causes
---
2015/10/25 20:58:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "causes".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('causes')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/causes
---
2015/10/25 20:58:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{block.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{block.source}...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bblock.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/causes
---
2015/10/25 20:58:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "causes".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('causes')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/causes
HTTP_REFERER=http://ikt-straubing.dev/causes
---
2015/10/25 20:58:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/25 20:58:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                2015/10/25 21:04:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2015/10/25 21:04:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/25 21:04:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2015/10/25 21:04:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/25 21:04:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/25 21:05:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/25 21:05:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "causes".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('causes')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/causes
HTTP_REFERER=http://ikt-straubing.dev/causes
---
2015/10/25 21:05:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/25 21:05:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/25 21:05:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/25 21:08:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "causes".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('causes')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/causes
HTTP_REFERER=http://ikt-straubing.dev/causes
---
2015/10/25 21:15:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/25 21:15:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/25 21:16:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/25 21:17:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/25 21:20:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "event".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('event')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/event
---
2015/10/25 21:20:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/25 21:22:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "event".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('event')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/event
---
2015/10/25 21:22:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/25 21:22:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/25 21:33:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "causes".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('causes')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/causes
HTTP_REFERER=http://ikt-straubing.dev/causes
---
2015/10/26 15:48:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2015/10/26 15:48:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 15:48:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2015/10/26 15:48:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 15:49:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "causes".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('causes')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/causes
HTTP_REFERER=http://ikt-straubing.dev/causes
---
2015/10/26 15:50:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 15:55:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/26 15:55:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/26 16:19:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 16:19:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 16:20:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 16:20:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 16:44:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 16:44:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 16:44:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 16:44:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 16:44:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 16:44:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 16:44:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about-us".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about-us')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about-us
---
2015/10/26 16:44:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 16:44:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 16:44:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/26 16:44:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/26 16:44:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/26 16:44:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/26 16:45:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 16:46:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about-us".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about-us')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about-us
---
2015/10/26 16:46:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 16:46:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 16:47:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about-us".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about-us')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about-us
---
2015/10/26 16:53:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 16:53:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 16:53:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 16:53:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 16:54:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 16:54:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 16:54:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about-us".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about-us')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about-us
---
2015/10/26 16:54:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 16:54:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 16:59:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about-us".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about-us')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about-us
---
2015/10/26 16:59:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 17:00:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about-us".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about-us')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about-us
---
2015/10/26 17:00:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 17:01:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about-us".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about-us')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about-us
---
2015/10/26 17:01:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 17:01:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/26 17:02:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 17:02:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about-us".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about-us')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about-us
---
2015/10/26 17:02:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 17:02:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 17:03:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about-us".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about-us')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about-us
---
2015/10/26 17:03:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 17:04:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about-us".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about-us')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about-us
---
2015/10/26 17:04:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 17:04:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 17:05:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about-us".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about-us')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about-us
---
2015/10/26 17:05:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 17:08:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/26 17:08:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 17:08:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 17:08:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 17:08:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "causes".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('causes')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/causes
HTTP_REFERER=http://ikt-straubing.dev/causes
---
2015/10/26 18:51:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 18:51:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 18:51:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 18:51:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 18:55:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/26 18:55:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/26 18:55:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/26 18:55:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/26 18:58:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/26 18:58:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/26 18:58:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/26 18:58:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/26 18:59:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 18:59:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 19:03:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "home".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('home')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/home
---
2015/10/26 19:03:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 19:03:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 19:05:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/26 19:05:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/26 19:05:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/26 19:05:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/editor_cigarrita_worker
---
2015/10/26 19:50:39 [error] [exception.CException] exception 'CException' with message 'Property "Configuration.id_facebook_page" is not defined.' in /var/www/html/ikt/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/html/ikt/yii/framework/db/ar/CActiveRecord.php(143): CComponent->__get('id_facebook_pag...')
#1 /var/www/html/ikt/protected/commands/facebookCommand.php(10): CActiveRecord->__get('id_facebook_pag...')
#2 /var/www/html/ikt/yii/framework/console/CConsoleCommandRunner.php(67): facebookCommand->run(Array)
#3 /var/www/html/ikt/yii/framework/console/CConsoleApplication.php(91): CConsoleCommandRunner->run(Array)
#4 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CConsoleApplication->processRequest()
#5 /var/www/html/ikt/cigarrita(12): CApplication->run()
#6 {main}
---
2015/10/26 19:51:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 19:51:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 19:51:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2015/10/26 19:51:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2015/10/26 20:07:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:07:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:16:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "home".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('home')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/home
---
2015/10/26 20:16:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:16:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:16:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:16:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:16:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/26 20:16:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/26 20:17:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:17:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:19:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about-us".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about-us')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about-us
---
2015/10/26 20:19:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:19:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:19:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about-us".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about-us')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about-us
---
2015/10/26 20:19:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:19:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:20:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about-us".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about-us')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about-us
---
2015/10/26 20:20:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:20:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:20:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/26 20:20:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/26 20:20:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:20:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:20:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:20:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:20:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:20:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:20:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/26 20:20:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/26 20:22:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "event".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('event')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/event
---
2015/10/26 20:22:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/26 20:22:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/26 20:22:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:22:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:22:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about-us".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about-us')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about-us
---
2015/10/26 20:22:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:22:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:23:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about-us".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about-us')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about-us
---
2015/10/26 20:23:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:23:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:32:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:32:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:32:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:32:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:32:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:32:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:33:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:33:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:34:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/event
---
2015/10/26 20:38:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gallery".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gallery')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/gallery
---
2015/10/26 20:38:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/about-us
---
2015/10/26 20:38:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:38:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.cover.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.cover.so...')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.cover.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:38:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "{{post.source}}".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('{{post.source}}')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/%7B%7Bpost.source%7D%7D
HTTP_REFERER=http://ikt-straubing.dev/home
---
2015/10/26 20:39:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "causes".' in /var/www/html/ikt/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/ikt/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('causes')
#1 /var/www/html/ikt/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /var/www/html/ikt/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/causes
HTTP_REFERER=http://ikt-straubing.dev/causes
---
